selfservice/hook/web_hook.go

```diff
@@ -557,9 +576,7 @@ func (e *WebHook) parseWebhookResponse(resp *http.Response, id *identity.Identit
 			Identity *identity.Identity `json:"identity"`
 		}
 
```

Comment:
Removed comments below. The panic was caused by the attempt to read a stream that was already read (line 547).
We must keep the forked approach to be able to log the content of webhook responses.